lib/canary/plugs.ex

```diff
@@ -181,6 +181,22 @@ defmodule Canary.Plugs do
   plug :load_resource, model: Post, id_name: "slug", id_field: "slug", only: [:show], persisted: true
   ```
   """
+  def authorize_controller(conn, opts) do
```

Comment:
Just 2 more comments. Sorry for the repeated change requests, but it's better if we make sure everything is fine now, rather than having to fix bugs later :)